etl/steps/data/garden/biodiversity/2023-01-11/cherry_blossom.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.helpers import Names
+from etl.paths import DATA_DIR
+
+log = get_logger()
+
+# naming conventions
+N = Names(__file__)
```

Comment:
Instead of `Names` you could use `PathFinder`, but you can just leave it for the next update if you prefer.